packages/x-components/src/x-modules/related-prompts/components/related-prompt.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="x-related-prompt x-bg-neutral-10 x-p-24 x-flex x-flex-col x-gap-16"
+    data-test="related-prompt"
+  >
+    <div class="x-related-prompt__info x-flex x-flex-col x-gap-16">
+      <slot name="header" :suggestionText="relatedPrompt.suggestionText">
+        {{ relatedPrompt.suggestionText }}
+      </slot>
+      <slot name="next-queries" :nextQueries="relatedPrompt.nextQueries">
+        <SlidingPanel :resetOnContentChange="false">
+          <div class="x-flex x-gap-8 x-pr-8">
+            <button
+              v-for="(nextQuery, index) in relatedPrompt.nextQueries"
+              :key="index"
+              @click="onClick(nextQuery)"
+              class="x-button x-button-lead x-button-sm x-button-outlined x-rounded-sm x-border-lead-50 x-text-neutral-75 hover:x-text-neutral-0 selected:x-text-neutral-0 selected:hover:x-bg-lead-50"
```

Comment:
Same here. Probably is better to have a prop to pass the desired classes. This way is easier from the outside customize the UI